### ISSUE TITLE
Sync of uniques not working - not downloading any data

### ISSUE THREAD

**Issue by Eccenux** [IITC-CE/ingress-intel-total-conversion#165]
So I tested the new CE edition plugin (version 0.3.1.20190315.122355).

1. Installed the plugin on desktop.
2. Installed the plugin on my phone (FF mobile).
3. Authorized the app with Google.
4. Reloaded both phone and desktop.
5. Checked uniques.

**Results**:
Portal that was red (uncaptured) on desktop remained uncaptured.

No messages was shown by the plugin. I only seen something about successfully creating a folder and that's it. After reload it only shows "Authorized".

So basically it seems not to be working at all.

**Expected**:
I expected sync to synchronize stuff between devices/browsers. Also the existing stuff.

The old version show something like "loading uniques", "replaced by other" etc. So some information about synchronizing stuff.

**Comment by modos189** [comment]
I'll fix it. Thanks.

**Comment by jwadamson** [comment]
sorry, not sure if same, but my uniques data in drive was just wiped out in drive by one of the instances. The file is a basic/empty dataset now.  Don't know what happened.

**Comment by AlfonsoML** [comment]
I've been testing a little the Sync plugin to add support for it in the "Pogo-s2" plugin (https://gitlab.com/AlfonsoML/pogo-s2/commit/dbc7354413ea324c281f0048b891a92c65ea9da5) and first I tested the bookmarks and uniques integration and although in the end it seemed to work, initially it was a little "buggy", it wasn't synchronizing and I think that it started correctly after doing some changes on both devices to uniques/bookmarks.  

So if there's something that can be improved that would be great. Testing with my plugin, I think that when the "non-modified" device gets the update, it keeps on reloading the whole data instead of noticing that the last-update-uuid hasn't been modified since last synchronization

**Comment by modos189** [comment]
@Eccenux @AlfonsoML please check Sync in test-builds

**Comment by johnd0e** [comment]
Fixed by d4846a6296c7b538001c51df984800c130a09c57(#212)
